src/utils.ts

```diff
@@ -57,7 +57,7 @@ export function isTruthy(value: string): boolean {
 
 export function shortAddress(address?: string | null | undefined, chars = 4): string {
   const addr = (address || '').toUpperCase();
-  return `0x${addr.substr(2, chars)}...${addr.substr(-chars)}`;
+  return `0x${addr.substr(2, chars - 1)}...${addr.substr(-chars)}`;
```

Comment:
On my vscode it says substr is deprecated and use substring() whenever possbile.
I tried this code and it wasnt giving me warning, displaying same address with substr.
  return `0x${addr.slice(2, chars + 1)}...${addr.slice(-chars)}`;